mobly/base_test.py

```diff
@@ -545,8 +533,7 @@ def run(self, test_names=None):
         class_record.test_begin()
         # Setup for the class.
         try:
-            if self._setup_class() is False:
-                asserts.fail("Failed to setup %s." % self.TAG)
+            self._setup_class()
         except Exception as e:
             self.log.exception("Failed to setup %s.", self.TAG)
```

Comment:
_exec_func swallows the exception and returns False, but that return value is never used anywhere.

- Is that return still needed?
- Can we give _exec_func a better name to more clearly show that it swallows exceptions?